packages/d3fc-series/examples/bar.js

```diff
@@ -2,25 +2,32 @@ var width = 500;
 var height = 250;
 var container = d3.select('#bar-svg');
 
-var dataGenerator = fc.randomGeometricBrownianMotion()
-  .steps(50);
+var dataGenerator = fc.randomGeometricBrownianMotion().steps(50);
```

Comment:
As it stands we have a different code style in these example files. Your formatter seems to have been a bit over-zealous. Any chance you can match the existing style with your changes and revert non-functional formatting changes?

Reply:
Done.